src/content/registry/npm.js

```diff
@@ -1,12 +1,16 @@
 import { createParseCommand } from './shared';
 
-const npmInstall = /(npm|yarn)( -g)?( global)? (install|i|add|update) /;
+const npmInstall = /((npm|yarn)( -g)?( global)? (install|i|add|update))\s/;
```

Comment:
Why did you need to change it?
If there is a bug here, please add a test for it.

In this code, the unit tests are the only explanation for it.

Reply:
Reverted back, this was committed by mistake